Add CHANGELOG for 1.10.2 (#5449)
spiffe/spire: CHANGELOG.md

# Changelog

## [1.10.2] - 2024-09-03

### Added

- `http_challenge` NodeAttestor plugin (#4909)
- Experimental support for validating container image signatures through Sigstore selectors in the docker Workload Attestor (#5272)

> - Metrics to track event-based cache (#5411)

Reviewer: ```suggestion
- Metrics that help monitoring the event-based cache (#5411)
```

Reviewer 2: Maybe "Metrics that help monitor the" or "Metrics for monitoring the"?

### Changed

- Delegated Identity API to allow subscription by process ID (#5272)
- Agent Debug endpoint to count SVIDs by type (#5352)

> - Agent health check to report an unhealthy status until Agent SVID is secured (#5298)

Reviewer 2: ```suggestion
- Agent health check to report an unhealthy status until the Agent is attested (#5298)
```

- Small documentation improvements (#5393)

### Fixed

> - `aws_iid` NodeAttestor to properly handle network interfaces (#5300)

Reviewer: ```suggestion
- `aws_iid` NodeAttestor to properly handle multiple network interfaces (#5300)
```

> - Server configuration to correctly propagate `sql_transaction_timeout` (#5345)

Reviewer: ```suggestion
- Server configuration to correctly propagate the `sql_transaction_timeout` setting in the experimental events-based cache (#5345)
```
